packages/k6-tests/src/koko/030-search-for-filename.md

```diff
@@ -0,0 +1,77 @@
+## Description
+The `ssearch for filename` test is intended to see how the instance behaves when a user creates a configurable amount of files and folders and then searches for them.
```

Comment:
Maybe you can clearify what it searches for? All files? Or by content text? Or file name?

Reply:
done, but for the future, the starting line is more or less a "elevator pitch", does it make sense to explain the behavior in detail there? Every step is described below... but maybe the list is to technical and we really need to mention those details in the introduction.

What do you think? need some brainstorming advice here 😗